public/ucec_msk_2018/case_lists/cases_sequenced.txt

```diff
@@ -0,0 +1,6 @@
+cancer_study_identifier: ucec_msk_2018
+stable_id: ucec_msk_2018_sequenced
+case_list_name: Sequenced Tumors
+case_list_description: All (Next-Gen) sequenced samples (195 samples)
```

Comment:
@yichaoS From the paper, all 197 were sequenced not 195.

Reply:
just fixed